include/xtensor/xtensor_config.hpp

```diff
@@ -23,6 +23,25 @@
     #endif
 #endif
 
+// Define if the library is going to be using exceptions.
+#if (!defined(__cpp_exceptions) && !defined(__EXCEPTIONS) && !defined(_CPPUNWIND))
+#undef XTENSOR_DISABLE_EXCEPTIONS
+#define XTENSOR_DISABLE_EXCEPTIONS
+#endif
+
+// Exception support.
+#if defined(XTENSOR_DISABLE_EXCEPTIONS)
+#define XTENSOR_THROW(_, msg)            \
+    {                                    \
+      std::fprintf(stderr, "%s\n", msg); \
```

Comment:
Why not use the C++ form, i.e. `std::cerr << msg << std::endl;` ?

Reply:
No reason - I will change right now.

Reply:
done - that's cleaner.